components/support/nimbus-cli/src/cmd.rs

```diff
@@ -318,15 +332,42 @@ impl LaunchableApp {
         Ok(true)
     }
 
-    fn start_app(&self, reset_db: bool, payload: Option<&Value>, log_state: bool) -> Result<bool> {
+    fn open(&self, deeplink: Option<&String>) -> Result<bool> {
+        self.start_app(false, None, false, deeplink)
+    }
+
+    fn create_deeplink(&self, deeplink: Option<&String>) -> Result<Option<String>> {
+        if deeplink.is_none() {
+            return Ok(None);
+        }
+        let deeplink = deeplink.unwrap();
+        Ok(if deeplink.contains("://") {
+            Some(deeplink.clone())
+        } else if let Some(scheme) = match self {
+            Self::Android { scheme, .. } | Self::Ios { scheme, .. } => scheme,
+        } {
```

Comment:
This is a really cool way to unwrap multiple enum options' values into a single value, I haven't considered this before!